### DISCUSSION TITLE
Sorry for the stupid question, I'm not an IT person.  HOW TO INSTALL pip3 install -r requirements.txt??!!

### DISCUSSION THREAD

**Discussion by uhtykakoi** [DrPython3/MailRipV2#18] [General]
HOW TO INSTALL pip3 install -r requirements.txt??!!
HOW TO INSTALL pip3 install -r requirements.txt??!!
HOW TO INSTALL pip3 install -r requirements.txt??!!

**Comment by DrPython3** [comment]
Install Python first. Python includes PIP. Depending on how Python is installed use python instead of python3 and pip instead of pip3.
